elections/steering/2024/candidate-aojea.md

```diff
@@ -0,0 +1,51 @@
+driving features, mainly in the networking area, and working across SIGs to improve the reliability
+and the quality of the project.
+
+Today I'm a TL for SIG Network and SIG Testing, maintainer
+of [KinD](https://kind.sigs.k8s.io/) and some other projects like [Cloud Provider Kind](https://github.com/kubernetes-sigs/cloud-provider-kind)
+and [Kube Network Policies](https://github.com/kubernetes-sigs/kube-network-policies).
+
+My involvement in the project has been rewarding, both professionally and personally, as I've built
+strong relationships and had a lot of fun contributing.
+
+## What I'll do
+
+Kubernetes, despite its age, is still a relatively young project. I'm committed to its continued growth and evolution.
+
+As part of the Steering Committee, in addition to existing responsibilities, I would like to help on these aspects for the project sustainability and success:
+
+- Technical excellence: The project is organised and driven technically by [Special Interest Groups](https://github.com/kubernetes/community/blob/master/governance.md#sigs). We must ensure SIGs are healthy, able to drive new features without sacrificing quality or exceeding their complexity budget, have a clear contributor ladder that help members to grow their skills and take on more responsibility, ...
+- Community: The community is the heart and soul of the project, without the active involvement of volunteers, developers, users, and other stakeholders,
+this project would not be able to thrive. Inclusivity, onboard, mentoring and recognition are fundamental aspects we should focus on.
+- Operational sustainability: We must track key metrics (contributions, downloads, ... ) and actively engage with the community to gather feedback and ensure the project aligns with user needs, so we can identify and mitigate potential risks.
+
+## Resources About Me
+
+- [Manage Session Modern Load Balancing, Improving Application's Resource Availability and Performance - Antonio Ojea & Gerrit DeWitt, Google](https://sched.co/1R2s6) Kubecon NA 2023
+- [Demystifing IPv6 Kubernetes - Antonio Jose Ojea Garcia, Google & Fernando Gont, Yalo](https://sched.co/1HyWI) Kubecon EU 2023
+- [SIGs Aren’t Silos: A Case Study Into Solving Inter-Domain Problems In Kubernetes Development - Swetha Repakula, Google & Antonio Ojea Garcia, RedHat](https://sched.co/182E6) Kubecon NA 2022
+- [Kubernetes Advanced Networking Testing With KIND](https://kccnceu2021.sched.com/event/iE3g/kubernetes-advanced-networking-testing-with-kind-antonio-ojea-redhat) Kubecon EU 2021
+- [Deep Dive: KIND - Benjamin Elder & Antonio Ojea](https://kccncna19.sched.com/event/Uah7/deep-dive-kind-benjamin-elder-google-antonio-ojea-garcia-suse) Kubecon NA 2019
+- [Google Open Source Peer Bonus 2021](https://opensource.googleblog.com/2021/04/announcing-first-group-of-google-open-source-peer-bonus-winners.html)
+- [SIG-Network and SIG-Testing Award 2020](https://www.youtube.com/watch?v=XCRkzgMTaJU)
```

Comment:
@aojea, the youtube link here says `This video isn't available anymore`. Could you please recheck? Thanks!

Reply:
I will remove the link if you don't mind, I can not find it now :)

Reply:
found other links more useful, replaced